lib/everypolitician/popolo/event.rb

```diff
@@ -38,8 +38,21 @@ def wikidata
       attr_reader :document
     end
 
+    class EventFactory < EntityFactory
+      subclasses [Election, LegislativePeriod]
+      default_class Event
+    end
+
     class Events < Collection
-      entity_class Event
+      entity_class EventFactory
+
+      def elections
+        where(classification: 'general election').sort_by(&:start_date)
```

Comment:
if Events are already pre-sorted in `popolo.rb`, do we need to also sort here (and in `legislative_periods` below)?